packages/cryptoassets/data/esdt.js

```diff
@@ -0,0 +1,5 @@
+module.exports = [
+  ['USDC', '555344432d633736663166', 18, '3045022100f03b1ec0d83d8d75d436f688c2007ffb0c71b39a4f72aba036062ea8d7ade99402203449426b83203617279c04ccb00387a77deffc468c27015bfc1590f1950edde9', "WrappedUSDC"],
+  ['MEX', '4d45582d343535633537', 18, '3044022076b7216cc07b827d5dec0468db31d9393b0028dcd5c81bcdc4f5371e0386061f022013f51d51c1421fbf67771a1edc1f607d750cf366392c5c09216fecb61e350f73', 'MEX'],
+  ['RIDE', '524944452d376431386539', 18, '3045022100afa25bb834c8a2a6e7f6e15f3f318e9da3b7991f2a3f9c2f58d73c700be08736022066d3d0ef00e754e050a4b1ba8455129e83dd01c2a8a6b79738373a463d10fe82', 'holoride'],
+]
```

Comment:
are these tokens understood by the device? in clear sign i mean

Reply:
Yes, all of these tokens are supported by ledger device already.